CONTRIBUTING.md

```diff
@@ -33,37 +33,54 @@ as well as other packages in the monorepo.
 ### Create a new lerna package
 
 ```shell
-$ lerna create truffle-mycmd
+$ lerna create mycmd
 ```
 
 ### Add the package to `@truffle/core`
 
 ```shell
-$ lerna add truffle-mycmd --scope=@truffle/core
+$ lerna add mycmd --scope=@truffle/core
 ```
 
 ### Create a new command in `@truffle/core`
 
-Create a new file in `packages/core/lib/commands/`, let's call it `mycmd.js`.
+1. Create a new directory in `packages/core/lib/commands/`, let's call it `mycmd`.
+2. Create 3 javascript files inside the **mycmd** directory with the filenames **run.js**, **meta.js** and **index.js**:  
```

Comment:
A new command needs to be explicitly listed in this [index.js](https://github.com/trufflesuite/truffle/blob/develop/packages/core/lib/commands/index.js) as per #5114. Maybe that file should have a comment to that point?

Reply:
@cds-amal I believe that that is already mentioned in the file:

### Link it from the commands/index.js file

```diff
--- packages/core/lib/commands/index.js
+++ packages/core/lib/commands/index.js
@@ -1,4 +1,5 @@
 module.exports = {
+  mycmd: require("./mycmd"),
```

However there is also the packages/core/lib/commands/commands.js file in the same directory. Do we need to write it in there as well?